cura/Settings/ProfilesModel.py

```diff
@@ -53,7 +53,7 @@ def getInstance(cls) -> "ProfilesModel":
     def _fetchInstanceContainers(self):
         global_container_stack = Application.getInstance().getGlobalContainerStack()
         if global_container_stack is None:
-            return []
+            return {}, {}

```

Comment:
Document what and why two dicts.

Reply:
The return value is documented in the inherited documentation. I've documented the other return statement here: 097cc72d89882b0bb05fb2627f91f5d36dbf7042